subprojects/docs/src/samples/userguide/dependencyManagement/workingWithDependencies/iterateArtifacts/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,23 @@
+repositories {
+    jcenter()
+}
+
+val scm by configurations.creating
+
+dependencies {
+    scm("org.eclipse.jgit:org.eclipse.jgit:4.9.2.201712150930-r")
+    scm("commons-codec:commons-codec:1.7")
+}
+
+// tag::iteration-task[]
+tasks.create("iterateResolvedArtifacts") {
+    val scm = configurations.getByName("scm")
```

Comment:
:x: `scm` is already in scope, this line is not needed and can be removed

Reply:
I chose to use this for two reasons:

 - the extracted sample in the documentation doesn't show the definition of scm, so the reader wouldn't have any idea what scm is
 - I could avoid the declaration of the val, but since the scm configuration is used twice in the block, that would force me to violate DRY and use configurations.getByName("scm") twice

But I realiee I could at least use 

    val scm = configurations[scm]

I'll use that. Tell me if it bothers you.

Reply:
Very good points!